easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.5.1b-intel-2018a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'RAxML-NG'
+version = '0.5.1b'
+
+homepage = 'https://github.com/amkozlov/raxml-ng'
+description = """RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality
+    criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and
+    Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/v%(version)s']
+sources = ['%(namelower)s_v%(version)s_linux_x86_64_MPI.zip']
```

Comment:
This is a tarball with pre-built binaries, shouldn't we download the `%(version)s.tar.gz` tarball instead?

Reply:
You are right. I think I should rename the package at least. I'm not sure if it would make sense to have an `easyconfig` file for the version with prebuilt binaries and one that builds the software completely from source with additional dependencies. For the latter case I have to look into `libpll` and create a easyconfig file for that first.

What would you prefer?

Reply:
We generally strongly prefer building from source, since then we can target the underlying architecture, which usually significantly benefits performance.

Reply:
@boegel Hi. Sorry for the late reply. 
The last time I checked it, I had problem fulfilling the dependencies properly. Some library part was rewritten and some required module did not have any release. I will see if this has changed when I find the time.